charts/producer-app-cleanup-job/README.md

```diff
@@ -0,0 +1,12 @@
+# producer-app cleanup helm chart
+
+This chart can be used to deploy a cleanup job for your Kafka producer app developed using streams-bootstrap.
+
```

Comment:
Maybe elaborate a bit, [cf.](#r888737919)

Reply:
What would you expect here?

Reply:
Only what you just added...